docs/cse/automation-service/about-automation-service.md

```diff
@@ -0,0 +1,171 @@
+* Entities
+   1. Open an Entity.
+   1. Click **Automations** under the Entity's name. 
+   1. Select an option under **Entity Automation**. 
+
+
+## Prerequisites
+
+### Configure role capabilities
+
+After the Automation Service is enabled for your organization, access to the Automation Service is controlled by [role capabilities](/docs/manage/users-roles/roles/role-capabilities) in the Sumo Logic platform. To get access to the Automation Service:
+1. In the left navigation bar of Sumo Logic, select **Administration > Users and Roles**.
+1. Click the **Roles** tab. 
+1. Click **Add Role** to create a new role for users of the Automation Service. Alternatively, you can select an existing role in the **Roles** tab and click **Edit**.
+1. Add the following capabilities:
+   * Cloud SIEM Enterprise
+     * Configuration
+       * View Automations
+       * Manage Automations
+       * Execute Automations
+   * Cloud SOAR
+     * View Cloud SOAR
+     * Automation Playbooks
+       * Access
+       * Configure
+1. Follow the directions to [access the Automation Service](#access-the-automation-service) to verify that you can see the **Automation** option in the **Configuration** menu.
+
+:::note
+To interact with most of the Automation Service features, you must have at least View Automations, View Cloud SOAR, and Access Playbooks permissions.
+:::
+
+### Authorize integrations
+
+To use [integrations](/docs/cse/automation-service/automation-service-integrations), you must authorize their resources for use in the Automation Service.
+1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
+1. Under **Integrations**, select **Automation**.<br/><img src={useBaseUrl('img/cse/automations-config-menu.png')} alt="Automation menu option" width="150"/>
+1. Click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/> 
+1. Click **Integrations** in the left navigation bar.
+1. Select the integration whose resource you want to authorize.
+1. Hover over the resource name and click the **Edit** button that appears.<br/><img src={useBaseUrl('img/cse/automations-edit-resource.png')} alt="Edit a resource" width="800"/> 
+1. Enter the authorization needed by the resource. What you enter is specific to the resource you're using. Each resource's configuration screen may be different, but in most cases, you will need information such as IP addresses, API tokens, usernames, and passwords for the application you're integrating with. For example, in the following screen enter the the **API URL** and **API Key**. <br/><img src={useBaseUrl('img/cse/automations-edit-resource-2.png')} alt="Edit a resource" width="400"/> 
+1. Click **Save** to save the configuration. 
+
+
+### API and Terraform support
+
+The [CSE API](/docs/cse/administration/cse-apis/) has been updated to support automations. The new endpoints include:
```

Comment:
I'm kind of wary of using "new / updated" language, unless it's in a release note. In six months, it won't be new anymore.

Reply:
Agreed. Fixed now.